libs/ui/src/lib/record-thumbnail/record-thumbnail.component.spec.ts

```diff
@@ -0,0 +1,24 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing'
+
+import { RecordThumbnailComponent } from './record-thumbnail.component'
+
+describe('RecordThumbnailComponent', () => {
+  let component: RecordThumbnailComponent
+  let fixture: ComponentFixture<RecordThumbnailComponent>
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      declarations: [RecordThumbnailComponent],
+    }).compileComponents()
+  }))
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(RecordThumbnailComponent)
+    component = fixture.componentInstance
+    fixture.detectChanges()
+  })
+
+  it('should create', () => {
+    expect(component).toBeTruthy()
+  })
```

Comment:
You should add some tests.
- img tag with correct url
- no img if no url

at least